packages/code-studio/src/main/AppInit.tsx

```diff
@@ -100,70 +100,6 @@ function AppInit(props: AppInitProps) {
   const plugins = usePlugins();
   // General error means the app is dead and is unlikely to recover
   const [error, setError] = useState<unknown>();
-  // Disconnect error may be temporary, so just show an error overlaid on the app
-  const [disconnectError, setDisconnectError] = useState<unknown>();
-  const [isFontLoading, setIsFontLoading] = useState(true);
-
```

Comment:
Why are all these lines removed here? Where did they go?

Reply:
Ok I'll check the full diff. I normally use the "changes since last review" option when I re-review